lib/package_provider/repository.rb

```diff
@@ -31,7 +31,7 @@ def self.temp_prefix=(tp)
     end
 
     def self.temp_prefix
-      @temp_prefix || 'pp_repo_'
+      @temp_prefix || "#{Process.pid}_instance_pid_pp_repo_"
```

Comment:
Well, I think there should be "pid" word because nobody knows what that number would mean if he/she see "package_repo_1555". So maybe we can go with "package_repo_pid#{Process.pid}"